tests/integration/redis-benchmark.tcl

```diff
@@ -126,6 +126,14 @@ start_server {tags {"benchmark network external:skip logreqres:skip"}} {
             r config set maxclients $original_maxclients
         }
 
+        test {benchmark: read last argument from stdin} {
+            set base_cmd [redisbenchmark $master_host $master_port "-x -n 10 set key"]
+            set cmd "printf arg | $base_cmd"
+            common_bench_setup $cmd
+            set cli_cmd [rediscli $master_host $master_port {get key}]
+            assert_equal "arg" [exec {*}$cli_cmd]
+        }
```

Comment:
```suggestion
            r get key
        } {arg}
```

Reply:
i see the test right above us is using `r` (for CONFIG`) and the one below us is using `cmdstat` (which uses `r`).
AFAICT this test spins up a normal test server and `r` is usable.
we pass `master_host` and `master_port` to the benchmark so it accesses that server.
bottom line, it should work.